setup.py

```diff
@@ -3,9 +3,12 @@
 with open("README.md", "r") as fh:
     long_description = fh.read()
 
+with open('requirements.txt') as f:
+    requirements = f.read().strip().split('\n')
+
```

Comment:
should this be removed? requirements.txt doesn't exist any longer

Reply:
Yes, can be removed! I added it just today to explore and it went nowhere.

Reply:
removed